src/lib/runner.ts

```diff
@@ -48,13 +50,32 @@ export default class Runner {
 			...this.rounds.all.map(ms => formatTime(ms)),
 		];
 	}
+
+	/** Creates a new runner from an json-like object */
+	static fromJSON(run: Run, json: ReturnType<Runner['toJSON']>) {
+		const runner = new Runner(run, json.id, json.name, json.alias);
+		runner._rounds = RoundManager.fromJSON(json.rounds);
+		return runner;
+	}
+
+	/** Converts this runner to a json-like object */
+	toJSON() {
+		return {
+			id: this.id,
+			name: this.name,
+			alias: this.alias,
+			rounds: this.rounds.toJSON(),
+		};
+	}
 }
 
 class RoundManager {
 	private _rounds: number[] = [];
 
+	constructor() {}
```

Comment:
Unused constructor should be removed again

Reply:
resolved: https://github.com/tametsi/ligas/pull/1/commits/ec83176d8b098346637230f754fca10aa994dab4